source/jormungandr/tests/proxy_realtime_cleverage_integration_tests.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
+# Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+import mock
+from jormungandr.tests.utils_test import MockRequests
+from tests.check_utils import get_not_null
+from .tests_mechanism import AbstractTestFixture, dataset
+
+MOCKED_PROXY_CONF = [
+    {
+        "object_id_tag": "KisioDigital",
+        "id": "KisioDigital",
+        "class": "jormungandr.realtime_schedule.cleverage.Cleverage",
+        "args": {
+            "destination_id_tag": "KisioDigital",
+            "timezone": "UTC",
+            "service_url": "http://XXXX",
+            "timeout": 15,
+            "service_args": {
+                "serviceID": "X",
+                "EntityID": "XX",
+                "Media": "XXX"
+            }
+        }
+    }
+]
+
+
+def _get_schedule(sched, sp_uri, line_code):
+    """ small helper that extract the information from a route point stop schedule """
+    return [
+        {'rt': r['data_freshness'] == 'realtime', 'dt': r['date_time']}
+        for r in next(rp_sched['date_times']
+                      for rp_sched in sched['stop_schedules']
+                      if rp_sched['stop_point']['id'] == sp_uri
+                      and rp_sched['route']['line']['code'] == line_code)
+        ]
+
+
+@dataset({'multiple_schedules': {'proxy_conf': MOCKED_PROXY_CONF}})
+class TestCleverageSchedules(AbstractTestFixture):
+    """
+    integration tests for cleverage
+
+    """
+    query_template = 'stop_points/{sp}/stop_schedules?data_freshness=realtime&_current_datetime=20160102T0800'
+
+    def test_stop_schedule_with_realtime_only(self):
+        """
```

Comment:
can you add some general comments on the test ? (even if the name is quite clear ?)

else you can remove the empty `""" """` :smile: